src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.3
+---
+
+ * Add `DomCrawlerAssertionsTrait::assertSelectorCount(string $selector, int $count)`
```

Comment:
shouldn't we swap arguments? the expected value is usually first

Reply:
I agree